Wrapping/Generators/Python/Tests/extras.py

```diff
@@ -163,6 +163,14 @@ def custom_callback(name, progress):
 image = itk.imread(filename, imageio=itk.PNGImageIO.New())
 assert type(image) == itk.Image[itk.RGBPixel[itk.UC], 2]
 
+# Python functional interface that determines the filter type
+# based on the primary input (dispatch)
+image = itk.imread(filename, itk.UC)
+# positional argument
+filtered = itk.median_image_filter(image)
+# required primary named input argument
+filtered = itk.median_image_filter(primary=image)
```

Comment:
Thanks for including snake case + keyword argument with your test, Matt! I'm still missing the "assert" here, though. It should test that a function call having a keyword argument produce the same output as the equivalent function call having a positional argument, right?  I'm thinking, something like the following:

        assert_equal_image(
            itk.median_image_filter(primary=input_image),
            itk.median_image_filter(input_image),
        )

Pablo (@phcerdan) already suggested how to write such an `assert_equal_image`, at https://discourse.itk.org/t/how-to-assert-that-images-are-equal-in-python-unit-test/7302

Reply:
Yes, we could also add that assertion. :+1: 

I added the comparison with an explanation for the additional checks we get in the Discourse thread.